src/js/components/InfiniteScroll/__tests__/InfiniteScroll-test.js

```diff
@@ -75,4 +80,22 @@ describe('InfiniteScroll', () => {
     );
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test(`Show item should be visible in window`, () => {
+    const { container } = render(
+      <Grommet>
+        <InfiniteScroll
+          items={simpleItems(300)}
+          // show={105}
+        >
+          {item => <Box key={item}>{item}</Box>}
+        </InfiniteScroll>
+      </Grommet>,
+    );
+    // item(104) = 'item 105' because indexing starts at 0
+    const renderedItems = container.firstChild.children.item(10).outerHTML;
+    expect(renderedItems).toContain('item 11');
+    // const renderedItems = container.firstChild.children.item(104).outerHTML;
+    // expect(renderedItems).toContain('item 105');
```

Comment:
You have this "Currently this test is passing, but once the show prop is fixed lines 96 and 97 need to be deleted and lines 89, 98, and 99 uncommented for the test to work correctly." called out in the PR. Might be helpful to document inline as code comments as well so we don't forget.

```suggestion
          /* Uncomment the next 2 lines once fix for show is in place */
          // show={105}
        >
          {item => <Box key={item}>{item}</Box>}
        </InfiniteScroll>
      </Grommet>,
    );
    // item(104) = 'item 105' because indexing starts at 0.
    /* Delete the next 2 lines once fix for show is in place */
    const renderedItems = container.firstChild.children.item(10).outerHTML;
    expect(renderedItems).toContain('item 11');
    /* Uncomment the next 2 lines once fix for show is in place */
    // const renderedItems = container.firstChild.children.item(104).outerHTML;
    // expect(renderedItems).toContain('item 105');
```